content/about.tsx

```diff
@@ -0,0 +1,50 @@
+import { Icons } from "../components/Icons";
+
+/**
+Markdown supported for description
+*/
+
+export const ABOUT_CONTENT = {
+  WHO_WE_ARE: {
+    TITLE: "Who we are",
+    DESCRIPTION: `TLSNotary is an open-source protocol developed by the **Privacy and Scaling Exploration (PSE)** research lab of the Ethereum Foundation.
+    \nTLSNotary is not a new project; in fact, it has been around for **more than a decade**. In 2022, TLSNotary was rebuilt from the ground up in **Rust** incorporating state-of-the-art cryptographic protocols. This renewed version of the TLSNotary protocol offers enhanced security, privacy, and performance.
+    `,
+  },
+  HOW_IT_WORKS: {
+    TITLE: "How it works",
+    DESCRIPTION:
+      "TLSNotary is a protocol which allows people to export data from any web application and prove facts about it to a third-party in a privacy-preserving way by leveraging secure multi-party computation (MPC) to authenticate data communicated between a Prover and a TLS-enabled web server.",
```

Comment:
I think we should put a diagram here, like this one: https://docs.tlsnotary.org/intro.html
(Source: https://github.com/tlsnotary/docs-mdbook/blob/main/diagrams/overview_prover_verifier.drawio)

We could also use elements of this diagram instead of the geometric patterns below

Reply:
Can we eventually add the image of the diagram on Figma? ty!

Reply:
I don't have edit rights. This is an svg version:
![overview_prover_verifier drawio](https://github.com/tlsnotary/landing-page/assets/735288/319791b6-4846-4249-ae3b-9b65cf9a43e6)

Reply:
thank you! I added the image!